libcasr/src/asan.rs

```diff
@@ -411,5 +417,13 @@ mod tests {
             "/casr_tests/bin/load_fuzzer".to_string()
         );
         assert_eq!(stacktrace[19].offset, 0xb98663);
+
+        assert_eq!(stacktrace[20].address, 0x998b40);
+        assert_eq!(
+            stacktrace[20].function,
```

Comment:
Can you test that error is returned?

Reply:
In this case error is not returned because we consider that `(anonymous namespace)::decrypt_xl<unsigned char> > const&) /xlnt/er+0x426cbd)` is function name. We didn't find `module+offset` in stacktrace entry.